gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 distributionUrl=https\://services.gradle.org/distributions/gradle-8.8-bin.zip
-networkTimeout=10000
-validateDistributionUrl=true
```

Comment:
both properties were added by gradle wrapper update task and are present in our other components. why should they be removed?

Reply:
yes i am tryign to find oput what caused the build to fail .. mayb networktimeout is causing the functionla tetst cases to fail .. i just thought i woudl try it or revert back